Slider/Assets/Scripts/Sliders/Grid/AreaGrids/OceanGrid.cs

```diff
@@ -14,6 +14,7 @@ public class OceanGrid : SGrid
     //public LostGuyMovement lostGuyMovement;
     public OceanArtifact oceanArtifact; // used for the final quest to lock movement
     public GameObject treesToJungle;
+    public List<int> buoytiles = new List<int> {1, 3, 4, 8, 9};
```

Comment:
are these changes for fixing https://trello.com/c/7fUBCOll/250-buoy-puzzle-npc-dialogue-is-broke lol

Reply:
No, this is to clean up the code (but if it happened to fix all of that it would be cool)